Scripts/Mobiles/Normal/DreadSpider.cs

```diff
@@ -14,64 +14,59 @@ public DreadSpider()
             this.Body = 11;
             this.BaseSoundID = 1170;
 
-            this.SetStr(196, 220);
-            this.SetDex(126, 145);
+            this.SetStr(196, 250);
+            this.SetDex(126, 155);
             this.SetInt(286, 310);
 
             this.SetHits(118, 132);
 
-            this.SetDamage(5, 17);
+            this.SetDamage(15, 18);
 
             this.SetDamageType(ResistanceType.Physical, 20);
             this.SetDamageType(ResistanceType.Poison, 80);
 
-            this.SetResistance(ResistanceType.Physical, 40, 50);
-            this.SetResistance(ResistanceType.Fire, 20, 30);
-            this.SetResistance(ResistanceType.Cold, 20, 30);
+            this.SetResistance(ResistanceType.Physical, 40, 55);
+            this.SetResistance(ResistanceType.Fire, 20, 45);
+            this.SetResistance(ResistanceType.Cold, 20, 40);
             this.SetResistance(ResistanceType.Poison, 90, 100);
-            this.SetResistance(ResistanceType.Energy, 20, 30);
+            this.SetResistance(ResistanceType.Energy, 20, 40);
 
             this.SetSkill(SkillName.EvalInt, 65.1, 80.0);
             this.SetSkill(SkillName.Magery, 65.1, 80.0);
-            this.SetSkill(SkillName.Meditation, 65.1, 80.0);
             this.SetSkill(SkillName.MagicResist, 45.1, 60.0);
-            this.SetSkill(SkillName.Tactics, 55.1, 70.0);
+            this.SetSkill(SkillName.Tactics, 55.1, 75.0);
             this.SetSkill(SkillName.Wrestling, 60.1, 75.0);
+            this.SetSkill(SkillName.Poisoning, 80.0);
+            this.SetSkill(SkillName.DetectHidden, 50.0, 60.0);
+            this.SetSkill(SkillName.Necromancy, 20.0);
+            this.SetSkill(SkillName.SpiritSpeak, 20.0);
 
             this.Fame = 5000;
             this.Karma = -5000;
 
             this.VirtualArmor = 36;
 
-            this.PackItem(new SpidersSilk(8));
+            if (!this.Controlled)
```

Comment:
Creatures will never be controlled once they are constructed.  You can remove these checks.  Also, for the shadow wyrm and Pheonix, they were made tamable on EA with the new pet training publish, correct?  So, would the new pet training that is missing be an issue?